app/helpers/validation_helper.rb

```diff
@@ -131,6 +131,38 @@ module ValidationHelper # rubocop:todo Metrics/ModuleLength
     nil
   ].freeze
 
+  TIME_OPTIONS = {
```

Comment:
I thought about this approach too but the reason I didn't go with this approach is because storing these as numbers makes defining the `:within_preferred_contact_time` much simpler: https://github.com/SaraAlert/SaraAlert/blob/513-custom-report-time/app/models/patient.rb#L322

Reply:
Good point, seems like it would be a pain without the simple casting, works for me!